packages/client/src/app/challenge/views/ChallengeView.vue

```diff
@@ -13,6 +13,10 @@ onMounted(async () => {
   const { user } = storeToRefs(useAuthStore());
   await getChallenges(user.value.id);
 });
+
+const toChallengeProgressRoute = (id) => {
+  return { name: 'challenge_progress', params: { userId: id } };
```

Comment:
The param used in challenges is challengeId, not userId (as we fetch one challenge for possibly several users). Could you please adjust? Thanks!